### ISSUE TITLE
AttributeError: 'builtin_function_or_method' object has no attribute 'write'

### ISSUE THREAD

**Issue by symroe** [DemocracyClub/WhoCanIVoteFor#718]
https://sentry.io/organizations/democracy-club-gp/issues/2305658724/?referrer=github_plugin

```
AttributeError: 'builtin_function_or_method' object has no attribute 'write'
(3 additional frame(s) were not displayed)
...
  File "django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "parishes/management/commands/import_parish_council_elections.py", line 12, in handle
    importer.import_objects()
  File "core/mixins.py", line 60, in import_objects
    self.create_object(row=row)
  File "parishes/importers.py", line 30, in create_object
    return sys.stdout.write.write("No election id, skipping row\n")
```

**Comment by michaeljcollinsuk** [comment]
@symroe this was resolved in #714 https://github.com/DemocracyClub/WhoCanIVoteFor/pull/714/files#diff-c9dd805b2084afb1a2ef04ae5cf364fbc4f43c626cf49b244648f727e777b78bR32 was this deployed? Could one of the instances be on the old code somehow?

**Comment by symroe** [comment]
ah, `Environment: test` :D No worries then :+1:
